src/pykeen/evaluation/evaluator.py

```diff
@@ -557,6 +570,10 @@ def evaluate(
 
     # Prepare for result filtering
     if filtering_necessary or positive_masks_required:
+        # FIXME!!
+        # all_pos_triples = torch.cat([model.triples_factory.mapped_triples, mapped_triples], dim=0)
+        # if additional_filtered_triples is not None:
+        #     all_pos_triples = torch.cat([all_pos_triples, additional_filtered_triples], dim=0)
```

Comment:
due to the removal of the triples factor from the model in #405, this doesn't work anymore. @mberr and I converged on the same solution for passing additional filtered triples there, but for this PR, we need to find the right place to concatenate this value with the training triples

Reply:
@cthoyt Are you fixing this now? Looks like a big mess :D

Reply:
@cthoyt My tests fail now too. Can you please revert all your commits?